modules/ob-compatibility-support-matrix.adoc

```diff
@@ -19,10 +19,11 @@ The link:https://access.redhat.com/support/offerings/techpreview[Technology Prev
 [options="header"]
 |===
 
-| {builds-shortname} Version 2+| Component Version | Compatible
-Openshift Pipelines Version | OpenShift Version
+| {builds-shortname} Version 2+| Component Version | Compatible Openshift Pipelines Version | OpenShift Version | Support
```

Comment:
```suggestion
| {builds-shortname} Version 2+| Component Version | Compatible OpenShift Pipelines Version | OpenShift Version | Support
```
I won't hold up merge but can you fix "OpenShift Pipelines" capitalization at some point?